ants_test.go

```diff
@@ -268,13 +268,13 @@ func TestPurge(t *testing.T) {
 }
 
 func TestRestCodeCoverage(t *testing.T) {
```

Comment:
I believe you should add new unit tests for your new feature.

Reply:
sure, I will add.